server/services/bookingService.test.ts

```diff
@@ -141,6 +141,20 @@ describe('BookingService', () => {
         service.bookingsToTableRows(bookingsDepartingSoon, premisesId, 'departure'),
       )
 
+      expect(bookingClient.allBookingsForPremisesId).toHaveBeenCalledWith(premisesId)
+    })
+  })
+  describe('currentResidents', () => {
+    it('should return table rows of the current residents', async () => {
+      const currentResidents = bookingFactory.arrived().buildList(2)
+
+      const premisesId = 'some-uuid'
+      bookingClient.allBookingsForPremisesId.mockResolvedValue(currentResidents)
```

Comment:
Wonder if it's worth returning some non-arrived bookings here, so we know we're filtering?